test/SdkTests/SdkTests.csproj

```diff
@@ -0,0 +1,283 @@
+          DestinationFolder="$(TestExecutionDirectory)"
+          SkipUnchangedFiles="true"
+          />
+
+    <!-- Set up NuGet feeds -->
+    <PropertyGroup>
+      <GeneratedNuGetConfig>$(TestExecutionDirectory)\NuGet.config</GeneratedNuGetConfig>
+      <NugetConfigContents>
+        <![CDATA[
+<configuration>
+  <packageSources>
+    <add key="dotnet-core" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" />
+    <add key="dotnet-windowsdesktop" value="https://dotnetfeed.blob.core.windows.net/dotnet-windowsdesktop/index.json" />
+    <add key="aspnet-aspnetcore" value="https://dotnetfeed.blob.core.windows.net/aspnet-aspnetcore/index.json" />
+    <add key="aspnet-aspnetcore-tooling" value="https://dotnetfeed.blob.core.windows.net/aspnet-aspnetcore-tooling/index.json" />
+    <add key="aspnet-entityframeworkcore" value="https://dotnetfeed.blob.core.windows.net/aspnet-entityframeworkcore/index.json" />
+    <add key="aspnet-extensions" value="https://dotnetfeed.blob.core.windows.net/aspnet-extensions/index.json" />
+  </packageSources>
+</configuration>
+        ]]>
+      </NugetConfigContents>
+    </PropertyGroup>
+
+    <WriteLinesToFile File="$(GeneratedNuGetConfig)"
+                  Lines="$(NugetConfigContents)"
+                  Overwrite="true" />
+
+    <!-- Allow test list to be overridden with ToolTestName property -->
+    <ItemGroup Condition="'$(ToolTestName)' != ''">
+      <SdkTest Remove="@(SdkTest)" />
+      <SdkTest Include="$(ToolTestName)" />
+    </ItemGroup>
+
+    <ItemGroup>
+      <TestProjectToRun Include="$(MSBuildProjectFullPath)">
+        <AdditionalProperties>ToolTestName=%(SdkTest.Identity)</AdditionalProperties>
+      </TestProjectToRun>
+    </ItemGroup>
+    
+    <MSBuild Projects="@(TestProjectToRun)"
+             Targets="RunTestsAsTool"
+             BuildInParallel="true"
+             />
+  </Target>
+  
+  <Target Name="RunTestsAsTool">
+    <PropertyGroup>
+      <ToolCommandName>testSdk$(ToolTestName)</ToolCommandName>
+      <TestProjectName>Microsoft.NET.$(ToolTestName).Tests</TestProjectName>
+      <TestLocalToolFolder>$(ArtifactsTmpDir)$(ToolCommandName)\</TestLocalToolFolder>
+    </PropertyGroup>
+
+    <RemoveDir Directories="$(TestLocalToolFolder)" />
+    <MakeDir Directories="$(TestLocalToolFolder)" />
+
+    <MakeDir Directories="$(ArtifactsTestResultsDir)" />
+
+    <Exec Command="dotnet new tool-manifest"
+      WorkingDirectory="$(TestLocalToolFolder)"/>
+
+    <PropertyGroup>
+      <SdkTestPackageVersion>$(MicrosoftNETSdkPackageVersion)</SdkTestPackageVersion>
+      <SdkTestPackageFeed>https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json</SdkTestPackageFeed>
+      
+      <InstallToolCommand>dotnet tool install --local $(ToolCommandName)</InstallToolCommand>
+      
+      <InstallToolCommand>$(InstallToolCommand) --version $(SdkTestPackageVersion)</InstallToolCommand>
```

Comment:
I think the reason we set that in the dotnet/sdk repo was to avoid issues where the test tool nupkg had been updated but the version hadn't.  So we overrode the DOTNET_CLI_HOME in order to be able to delete all the cached information about the package.

In core-sdk, we are inserting the tests from another repo, so we shouldn't have clashes between different contents of a package with the same version number.